runtime/help/plugins.md

```diff
@@ -121,6 +121,15 @@ The packages and functions are listed below (in Go type signatures):
        current pane is not a BufPane.
 
     - `CurTab() *Tab`: returns the current tab.
+
+    - `Redraw()`: triggers a screen redraw. This is useful after 
+      asynchronous actions that result in changes in the UI
+
+    - `Lock()`: Access the Lua's lock mutex. This is useful to prevent 
```

Comment:
`Lock` is the mutex object, not a function, so the parentheses are misleading.

(My first thought was "so we expose Lock() but don't expose Unlock()?". Then I realized that `Lock` is the mutex itself, and is supposed to be used via `micro.Lock:Lock()` and `micro.Lock:Unlock()`.)

BTW I've found that it doesn't really work: #2945. Looks like no one ever even tried to use it.

Reply:
Your #2944 includes a bit more detailed documentation. Maybe you both can somehow merge it into one PR and close the duplication. There is no need to keep two of them open.
The `Lock()` documentation can be added to #2945, if you think that it then fits more there.

Reply:
Yep, no need to keep both open. I've closed #2944.

Reply:
I've used the documentation from your PR for Redraw and adjusted the Lock case.